### ISSUE TITLE
[Programmatic Access – Make Code - Blocks] Incorrect role is defined for 'pause extention, slow-mo' button control.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3630]
User Experience:
Users who depends upon screen reader get confused if the Screen reader reads incorrect role of any control. They'll not be able to know whether it is button or whether it is a non-interactive control. Users will understand it to be a interactive control as screen reader announces link while navigating in scan mode and hence it will not be a good usable experience for the users.

Note: User credentials should NOT be included in the bug.

Pre-Requisite:
Open F12 Developer tool.

Repro Steps:

Launch application URL
Home page will open.
Navigate "New Project" option and select it.
Editor screen will open.
Navigate to "Simulator" section appearing in left pane.
Navigate to "Toggle Debug Mode" option and select it.
Navigate through all newly appeared controls like Play/pause, Exit debug mode, Step over, step into, step out etc.
Verify the issue.

Actual Result:
Incorrect role 'Presentation' is defined for button control.

Expected Result:
Correct role should be define as 'Button' control present under screen.

Note:
Same issue exists on Firefox + NVDA & Chrome + JAWS.

WCAG Reference:
[WCAG 4.1.2 – Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html%7D)

Test Environments:
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG4.1.2, #Win10-Edge(Chromium)

![WCAG4 1 2_Incorrect role is define](https://user-images.githubusercontent.com/72871089/98531542-7c48e680-22a6-11eb-8845-45b2385a9cba.png)

**Comment by jwunderl** [comment]
The icons themselves are for presentation, the encapsulating button that is interactive does not have the presentation role. This is behaving as expected.
